src/main/java/teammates/common/util/Sanitizer.java

```diff
@@ -131,6 +131,16 @@ public static String sanitizeForHtml(String str) {
     }
 
     /**
+     * Escapes HTML tag safely. This function can be applied multiple times.
+     */
+    public static String sanitizeForHtmlTag(String str) {
+        if (str == null) {
+            return null;
+        }
+        return str.replace("&", "&amp;").replace("<", "&lt;").replace(">", "&gt;");
```

Comment:
Also, I don't think the issue "the content is not the original content" is relevant here. If it happens, let it. Otherwise you'll need to handle `&quot;`, `&middot;`, etc. `&lt;` and `&gt;` are special cases for obvious reason.